eva-accession-import-dbsnp2/pom.xml

```diff
@@ -59,6 +52,16 @@
             <groupId>uk.ac.ebi.eva</groupId>
             <artifactId>variation-commons-batch</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.3.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <scope>runtime</scope>
```

Comment:
This should be testing scope, right? Or are you using it at runtime too?

Reply:
Spring would require this at run-time to track job progress as well.